sub-logs/everyone.json

```diff
@@ -0,0 +1,17 @@
+{
+  "repo": "",
+
+  "owner": {
+    "username": "roktim32",
+    "email": "roktimsenapoty2@gmail.com"
+  },
+
+  "target": {
+    "CNAME": {
+       "name": "everyone", 
+       "value": "https://roktim32.github.io/everyone-is-a-good.dev/"
```

Comment:
CNAME should not have any protocols or /. An example of a cname would be: dev-dibster.github.io

Your first CNAME had the correct value, it just didn't display anything. If this site works on roktim32.github.io, then I can approve this pull request.

Reply:
i have fixed the CNAME part. As of now it wont show anything. Can you please merge roktim32.github.io?